src/button/Button.story.mdx

```diff
@@ -0,0 +1,237 @@
+import { useState } from 'react'
+import { action } from '@storybook/addon-actions'
+import { Meta, Preview, Props } from '@storybook/addon-docs/blocks'
+
+import { color } from '../_utils/branding'
+import { Button, ButtonStatus } from '../button'
+import { ArrowIcon } from '../icon/arrowIcon'
+
+export const commonProps = {
+  onClick: action('clicked'),
```

Comment:
😮 I didn't know that.

Reply:
I thought we couldn't use it anymore in mdx for some reasons.

Reply:
Well, it seems to be working